docs/source/evaluation_parameters.rst

```diff
@@ -0,0 +1,52 @@
+.. _evaluation_parameters:
+
+================================================================================
+Using Evaluation Parameters
+================================================================================
+
+Often, the specific parameters associated with an expectation will be derived from upstream steps in a processing \
+pipeline. For example, we may want to `expect_table_row_count_to_equal` a value stored in a previous step, but we \
+may still want to ensure that we can use the same expectation configuration object.
+
+Great Expectations makes working with parameters of that kind easy! When declaring an expectation, you can specify that \
+a particular argument is an evaluation parameter that should be substituted at evaluation time, and provide a temporary \
+value that should be used during the initial evaluation of the expectation.
+
+.. code-block:: python
+
+    >> my_df.expect_table_row_count_to_equal(
+        value: {"$PARAMETER": "upstream_row_count",
+                "$PARAMETER.upstream_row_count": 10}
+        result_format={'result_format': 'BOOLEAN_ONLY'}
+    )
+    {
+        'success': True
+    }
+
+
+You can also store parameter values in the meta object to be available to multiple expectations or while declaring \
```

Comment:
Should this be `evaluation_parameters` instead of `meta`?

Reply:
Agreed this isn't particularly clear, and it may be a legacy of the way we were thinking of this earlier as a meta thing. 

The current structure is like this:
```
_expectations_config: {
    ...
    meta: {
        ...
        evaluation_parameters: {
            param1: val1,
            ...
        }
    }
}
```

It seems like instead, I could just move evaluation_parameters out to be a top-level key in the _expectations_config object. Thoughts?

Even without that change, perhaps better wording for the docs would be "You can also store parameter values in a special dictionary called evaluation parameters that is stored in the meta object of the _expectations_config"

Reply:
A top-level key makes more intuitive sense to me.

"You can also store parameter values in a special dictionary called `evaluation_parameters` that is stored in the expectations_config"

Reply:
Updated.